compiler/main/runpasses.cpp

```diff
@@ -90,6 +90,9 @@ static PassInfo sPassList[] = {
   // Chapel to AST
   RUN(parse),                   // parse files and create AST
   RUN(checkParsed),             // checks semantics of parsed AST
+  RUN(docs),                    // if fDocs is set, this will generate docs.
```

Comment:
I don't think I need to, but it would make things clearer for future developers so I will.